test/origin_simulator/duration_test.exs

```diff
@@ -0,0 +1,85 @@
+defmodule OriginSimulator.DurationTest do
+  use ExUnit.Case
+
+  alias OriginSimulator.Duration
+
+  describe "parsing a integer" do
+    test "0" do
+      assert Duration.parse(0) == 0
+    end
+
+    test "1" do
+      assert_raise RuntimeError, "Invalid stage latency, please define time in s or ms", fn () -> Duration.parse(1) end
+    end
+  end
+
+  describe "parsing a binary string" do
+    test "0" do
+      assert Duration.parse("0") == 0
+    end
+
+    test "0ms" do
+      assert Duration.parse("0ms") == 0
+    end
+
+    test "0s" do
+      assert Duration.parse("0s") == 0
+    end
+
+    test "100ms" do
+      assert Duration.parse("100ms") == 100
+    end
+
+    test "1ms" do
+      assert Duration.parse("1s") == 1000
```

Comment:
can we do a test for what happens when we do: `Duration.parse("1.2s")`?

Reply:
done